plugins/scaffolder-react/src/next/components/Stepper/Stepper.tsx

```diff
@@ -149,7 +150,9 @@ export const Stepper = (stepperProps: StepperProps) => {
     props.onCreate(formState);
     const name =
       typeof formState.name === 'string' ? formState.name : undefined;
-    analytics.captureEvent('create', name ?? props.templateName ?? 'unknown');
+    analytics.captureEvent('create', name ?? props.templateName ?? 'unknown', {
+      value: props.minutesSaved,
```

Comment:
Should we subtract the amount of time it took to fill out the form and wait for the templating task to run? 😏